### PR TITLE
New Cloud App sample queries

### PR DESCRIPTION
**Pull request by Keithbfleming**
Required items, please complete
   
   Change(s):
   - Added 3 new Cloud App Sample queries to the community project

   Reason for Change(s):
   - Providing samples to MDA customers so they can get started using advanced hunting on the cloud app tables

   Version Updated:
   - 1.0.0.0 for all queries

   Testing Completed:
   - Tested in M365D advanced hunting

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @Keithbfleming, please add the properties for PerformedByDisplayName in CloudAppEvents Table here - https://github.com/Azure/Azure-Sentinel/blob/master/.script/tests/KqlvalidationsTests/CustomTables/CloudAppEvents.json

Also please add `relevanttechniques` in all the hunting queries, refer - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#relevanttechniques for the same.

File: Hunting Queries/Microsoft 365 Defender/Cloud Apps/aad-group-adds.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+id: f5e4d3c2-b1a0-4f9d-8e7c-6b5a4d3e2c1f
+name: AAD group adds in the last 7 days
+description: |
+  This query looks for AAD group adds identified by Microsoft Defender for Cloud Apps. It will require an corresponding app connector in Microsoft Defender for Cloud Apps.
```
Comment (v-rbajaj) [inline Hunting Queries/Microsoft 365 Defender/Cloud Apps/aad-group-adds.yaml:4]: Description character size is limited to 255 characters, please shorten it.

**Comment by v-rbajaj** [comment]
Hi @Keithbfleming, please provide some update on the above comments.

**Comment by Keithbfleming** [comment]
Updated the PR with the requested items.
________________________________

**Comment by v-rbajaj** [comment]
Hi @Keithbfleming, Thanks for updating these changes, you will get an update by 14 Sep 2023.

**Review by v-rbajaj — APPROVED** [review]
Looks good

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**